code-challenges.test.js

```diff
@@ -10,34 +31,61 @@
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
-// --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multiplied by 3.
-
-// a) Create a test with expect statements for each of the variables provided.
-
-const numbersArray1 = [6, 7, 8, 9, 10]
-// Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
-// Expected output: [72, 81, 90, 99, 108]
-
-
-// b) Create the function that makes the test pass.
-
+// --------------------Replaced the Instructor Example With the Template
 
+// // a describe method that lists the name of the function OR naming of the particular test.
+// describe("hello", () => {
+//     // a test/it method, nested within the describe block, that in plain words, describes that the function does.
+//     it("returns a string that says hi", () => {
+//       //an expect method, nested within the test block, calling on the hello() function, followed by the .toEqual() matcher that checks the expected output of the function return.
+//       expect(hello()).toEqual("hi");
+//     });
+//   });
 
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
+
 // a) Create a test with expect statements for each of the variables provided.
 
+
 const object1 = { number: 15 }
 // Expected output: "15 is divisible by three"
 const object2 =  { number: 0 }
 // Expected output: "0 is divisible by three"
 const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
 
+describe("divThree", () => {
+    it("determines if the number is divisible by 3 or not and tells you", () => {
+      expect(divThree(object1.number)).toEqual(`15 is divisible by three`);
+      expect(divThree(object2.number)).toEqual(`0 is divisible by three`);
+      expect(divThree(object3.number)).toEqual(`-7 is not divisible by three`);
+    });
+  });
+
+
+//   divThree
+//     ✕ determines if the number is divisible by 3 or not and tells you (2 ms)
+
+//   ● divThree › determines if the number is divisible by 3 or not and tells you
+
+//     ReferenceError: divThree is not defined
 
 // b) Create the function that makes the test pass.
 
+// Psuedo Code
+// Create a function named divThree
+// Input: take each variable and use the Modulo Method and see if its divisible by three (% 3)
+// Output: will return with a string telling you if the number is divisble by three or not
+
+
+const divThree = (number) => {
+    if (number % 3 === 0) {
+        return `${number} is divisible by three`
+    } else {
+        return `${number} is not divisible by three`
+    }
+}
 
```

Comment:
this structure is great. You will just have to use the dot notation from earlier. Ex:
`const divThree = (object) => {`
`    if (object.number % 3 === 0) {`
        return `${object.number} is divisible by three`
`    }`

Reply:
It is hard to portray the exact syntax in this comment section. Reach out if you need clarity on these explanations